fe/fe-core/src/main/java/com/starrocks/analysis/TaskName.java

```diff
@@ -0,0 +1,51 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
You need to use starrocks license header, this is license header for Apache Software Foundation's project.

Reply:
ok, this fixed in #24126